src/core/stack.js

```diff
@@ -351,6 +352,17 @@ export default class Stack {
         return this;
     }
 
+    /**
+     * @method Taxonomies
+     * @memberof Stack
+     * @description A method to set base url to taxonomies endpoint
+     * @returns {Stack}
+     */
+    Taxonomies() {
+        this.type = "taxonomy"
```

Comment:
@abhinav-from-contentstack It should return instance of taxonomy class

Reply:
done @ishaileshmishra